content/id/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,374 @@
+Kelompok konfigurasi yang disarankan untuk semua permintaan terbagi dalam lima prioritas
+kelas:
+
+* Tingkat prioritas `system` diperuntukkan bagi permintaan dari grup `system:nodes`,
+  mis. Kubelet, yang harus bisa menghubungi server API agar
+  mendapatkan _workload_ untuk dijadwalkan.
+
+* Tingkat prioritas `leader-election` diperuntukkan bagi permintaan dalam pemilihan pemimpin (_leader election_)
+  dari _controller_ bawaan (khususnya, permintaan untuk `endpoint`, `configmaps`,
+  atau `leases` yang berasal dari `system:kube-controller-manager` atau pengguna
+  `system:kube-scheduler` dan akun Service di _namespace_ `kube-system`). Hal ini 
+  penting untuk mengisolasi permintaan ini dari lalu lintas yang lain karena 
+  kegagalan dalam pemilihan pemimpin menyebabkan _controller_ akan gagal dan memulai kembali (_restart_), 
+  yang pada akhirnya menyebabkan lalu lintas yang lebih mahal karena _controller_ 
+  yang baru perlu menyinkronkan para informannya.
+
+* Tingkat prioritas `workload-high` diperuntukkan bagi permintaan yang lain dari _controller_ bawaan.
+  
+* Tingkat prioritas `workload-low` diperuntukkan bagi permintaan dari akun Service yang lain,
+  yang biasanya mencakup semua permintaan dari _controller_ yang bekerja didalam Pod.
+  
+* Tingkat prioritas `global-default` menangani semua lalu lintas lainnya, mis.
+  perintah interaktif `kubectl` yang dijalankan oleh pengguna yang tidak memiliki hak khusus.
+
+Kemudian, ada dua PriorityLevelConfiguration dan dua FlowSchema yang telah
+dibangun dan tidak mungkin ditimpa ulang:
+
+* Tingkat prioritas khusus `exempt` diperuntukkan bagi permintaan yang tidak akan dikenakan
+  _flow control_ sama sekali: permintaan itu akan selalu diteruskan sesegera mungkin.
+  FlowSchema `exempt` khusus mengklasifikasikan semua permintaan dari kelompok `system:masters`
+  ke dalam tingkat prioritas khusus ini. Kamu juga dapat menentukan FlowSchema lain yang mengarahkan
+  permintaan lain ke tingkat prioritas ini juga, apabila permintaan tersebut sesuai.
+  
+* Tingkat prioritas khusus `catch-all` digunakan secara kombinasi dengan spesial
+  FlowSchema `catch-all` untuk memastikan bahwa setiap permintaan mendapatkan proses
+  klasifikasi. Biasanya kamu tidak harus bergantung pada konfigurasi _catch-all_ ini,
+  dan kamu seharusnya membuat FlowSchema _catch-all_ dan PriorityLevelConfiguration kamu sendiri
+  (atau gunakan konfigurasi `global-default` yang sudah diinstal secara bawaan) secara benar.
+  Untuk membantu menemukan kesalahan konfigurasi yang akan melewatkan beberapa klasifikasi
+  permintaan, maka tingkat prioritas `catch-all` hanya wajib mengijinkan satu konkurensi
+  bersama dan tidak melakukan memasukkan permintaan dalam antrian, sehingga membuat lalu lintas
+  yang secara relatif hanya sesuai dengan FlowSchema `catch-all` akan ditolak dengan kode kesalahan HTTP 429.
+
+## Sumber daya (_Resources_)
```

Comment:
sumber daya is generally understandble, but if you think it's necessary to help adding "resource", use the singular form
```suggestion
## Sumber daya (_Resource_)
```

Reply:
Agree!